bookmaker_titlepage.rb

```diff
@@ -31,16 +32,32 @@ def testingValue(file, logkey='')
 end
 
 # wrapping a method from isbnFinder.rb so we can get output for json_logfile
-def getIsbns(logkey='')
-  pisbn, eisbn, allworks = findBookISBNs(Bkmkr::Paths.outputtmp_html, Bkmkr::Project.filename)
+def getIsbns(file, filename, logkey='')
+  pisbn, eisbn, allworks = findBookISBNs(file, filename)
   return pisbn, eisbn, allworks
 rescue => logstring
   return '','',''
 ensure
   Mcmlln::Tools.logtoJson(@log_hash, logkey, logstring)
 end
 
-def findImprint(pisbn, eisbn, logkey='')
+# Find any custom metadata overrides that users added to the file
+def getMetaElement(file, name, logkey='')
+  logstring = "none"
+  metaelement = File.read(file).match(/(<meta name="#{name}" content=")(.*?)("\/>)/i)
+  unless metaelement.nil?
+    metaelement = HTMLEntities.new.decode(metaelement[2])
+    logstring = metaelement
+  end
+  return metaelement
+rescue => logstring
+ensure
```

Comment:
For all the json logging, if there was a return, I usually returned an empty element of the same type in the rescue statement; eg at line 54 I would have a `return ''` 
If nothings returned when something is expected I think in throws an exception in the stderrlog, which is not a big deal, but I think if another method was called on that var it could cause issues..  I was really just trying to trap errors that might crash the script outright, so I added those as a blanket rule?